pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -59,14 +59,22 @@ let
         rev = "a059e7404a6db799f4da0ad696e65ae9c854b4b0";
       in
       {
-        version = "${branchName}-${lib.substring 0 8 rev}";
+        version = "${branchName}-${lib.substring 0 7 rev}";
 
         src = fetchgit {
           url = "https://github.com/qt/qtwebengine.git";
-          sha256 = "19l1i4sk3pvnwbvz5s97jchqawfz8k1xmjza29bgvkp1zz96r0jx";
+          sha256 = "0646fk8gg6lcsp1c50is9jcvr7yifn508709jgds90cnvzzb42mx";
           inherit rev branchName;
           fetchSubmodules = true;
           leaveDotGit = true;
+          name = "qtwebengine-${substring 0 7 rev}.tar.gz";
+          postFetch = ''
+            # remove submodule .git directory
+            rm -rf $out/src/3rdparty/.git
```

Comment:
```suggestion
            rm -r $out/src/3rdparty/.git
```

Reply:
.git is write protected.

Reply:
Generally ``rm -f`` should be avoided because it silently fails but doing chmod +w before is als not nice.